modules/account/src/test/java/edu/stanford/spezi/module/account/AccountEventsTest.kt

```diff
@@ -0,0 +1,75 @@
+package edu.stanford.spezi.module.account
+
+import com.google.common.truth.Truth.assertThat
+import edu.stanford.spezi.core.testing.runTestUnconfined
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.first
+import org.junit.Before
+import org.junit.Test
+
+class AccountEventsTest {
+
+    private lateinit var accountEvents: AccountEvents
+
+    @Before
+    fun setUp() {
+        accountEvents = AccountEvents(CoroutineScope(Dispatchers.Unconfined))
```

Comment:
no need for setup, you can declare directly once and make use of the `SpeziTestScope`. `private val accountEvents = AccountEvents(scope = SpeziTestScope())`. JUnit creates new test class instances before each test ([reference](https://www.baeldung.com/junit-testinstance-annotation#introduction))